dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -511,12 +511,44 @@
         <xsl:choose>
             <xsl:when test="//dspace:field[@mdschema='dc' and @element='type'][1]">
                 <xsl:variable name="types" select="tokenize(//dspace:field[@mdschema='dc' and @element='type'][1]/text(),';')"/>
-                <xsl:element name="resourceType">
-                    <xsl:attribute name="resourceTypeGeneral">
-                        <xsl:value-of select="$types[1]"/>
-                    </xsl:attribute>
-                    <xsl:value-of select="$types[2]" />
-                </xsl:element>
+
+                <!-- Check if the type is 'corpus' and handle it explicitly -->
```

Comment:
Why this?

Reply:
Because this is our own custom-defined type and it's not one of the default types provided by DataCite like Sounds, Chapters, etc., I could either use 'this' or set all custom-defined types to 'Others'. In my opinion, this was the better option, so I used it. Without this, I couldn't register the DOI because the metadata was unrecognized.